apps/docs/src/app/core/component-docs/layout-grid/examples/layout-grid-basic-example.component.html

```diff
@@ -1,25 +1,27 @@
 <fd-layout-grid>
-    <div [fdLayoutGridCol]="2">
-        <example-layout-grid-block>Size: 2</example-layout-grid-block>
-    </div>
+    <div fdLayoutGridRow>
```

Comment:
Can we add check in fd-layout-grid component that if there's no fdLayoutGridRow directive, we should wrap everything? Just to be backward-compatible with older versions?

Reply:
Good catch, breaking change removed.